recipe/meta.yaml

```diff
@@ -1,45 +1,51 @@
 {% set name = "xraylarch" %}
-{% set version = "0.9.50" %}
+{% set version = "0.9.55" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/xraypy/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: f64eee63bca13a67d313a06c9b437502ea59a5ca961075756ebf453e9dc1da81
-  patches:
-    - 0001-MNT-remove-the-desktop-shortcut-logic.patch
-    - 0002-MNT-remove-the-feff-wrappers-and-binaries.patch
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41fee953ea308c52ecf0de11140cb89d5b7475f1a67ca69e7c71c15697bf117e
 
 build:
-  number: 1
+  number: 0
+  entry_points:
+    - larch = larch.apps:run_larch
+    - larch_server = larch.apps:run_larch_server
+    - feff6l = larch.apps:run_feff6l
+    - feff8l = larch.apps:run_feff8l
+    - xas_viewer = larch.apps:run_xas_viewer
+    - gse_mapviewer = larch.apps:run_gse_mapviewer
+    - xrfdisplay = larch.apps:run_xrfdisplay
+    - xrfdisplay_epics = larch.apps:run_xrfdisplay_epics
   script: |
-    # We do not have a license to distribute built FEFF binaries.
```

Comment:
I still have deep concerns about this, but https://xraypy.github.io/xraylarch/ claims:

> The Feff6L and Feff8L codes are the open-source versions of Feff6 and Feff8, written by the Feff Project, and included with Larch by permission and with license to redistrubute.

but gives no indication of what the Feff license is or evidence that the feff authors agree. 

I _suspect_ that https://github.com/newville/ifeffit/blob/master/src/feff6/LICENSE is the license.

Reply:
In https://github.com/xraypy/xraylarch/issues/280, Matt pointed to https://github.com/xraypy/feff85exafs/blob/master/LICENSE_Feff6L.txt, so I think we are good.